src/js/shield_canvas_draw.js

```diff
@@ -215,22 +215,23 @@ export function trapezoid(
 
   let x0 = lineWidth;
   let x11 = width - lineWidth;
-  let y0 = angle > 0 ? lineWidth : CS - lineWidth;
-  let y3 = angle > 0 ? CS - lineWidth : lineWidth;
+  let y0 = shortSideUp ? CS - lineWidth : lineWidth;
+  let y3 = shortSideUp ? lineWidth : CS - lineWidth;
 
   let y1 = y0 + angleSign * drawRadius * (1 + sine);
   let y2 = y3 - angleSign * drawRadius * (1 - sine);
 
   let x1 = x0 + (y1 - y0) * tangent;
   let x2 = x1 + drawRadius * cosine;
-  let x3 = x0 + (y2 - y0) * tangent;
-  let x4 = x0 + (y3 - y0) * tangent;
-  let x5 = x3 + drawRadius * cosine;
+  let x3 = x0 + angleSign * (y2 - y0) * tangent;
+  let x4 = x0 + angleSign * (y3 - y0) * tangent;
+  let x5 = x3 + angleSign * drawRadius * cosine;
   // let x6 = width - x5;
```

Comment:
These were already here but is there a need for the commented code? and also x10 below.

Reply:
Technically not needed, but might be useful to whoever is reading this in the future to see the control points I calculated but ended up not needing. If I get rid of these commented lines, should I decrement the following variable names so no numbers are skipped?

Reply:
Whatever you think is the most consistent/easy to follow?  In general, commented-out code is a code smell so if there is any there should be a reason.

Reply:
Commented lines of code removed; variable names decremented.